src/main/java/org/zalando/intellij/swagger/traversal/path/openapi/MainPathResolver.java

```diff
@@ -86,7 +86,9 @@ public final boolean childOfSchema(final PsiElement psiElement) {
 
   @Override
   public final boolean isSchemaRefValue(final PsiElement psiElement) {
-    return hasPath(psiElement, "$.**.schema.$ref") || hasPath(psiElement, "$.**.items.$ref");
+    return hasPath(psiElement, "$.**.schema.$ref")
+        || hasPath(psiElement, "$.**.items.$ref")
+        || hasPath(psiElement, "$.components.schemas.**.$ref");
```

Comment:
Technically it's possible to reference almost everything. (e.g. a [property](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#simple-model), an [example](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#example-object-examples), [additionalProperties/allOf](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#model-with-mapdictionary-properties) etc.)

Does it make sence to change this to `$.**.$ref`?

Reply:
The reason I've used these specific paths is that the references point to specific types of objects. For example, we should not suggest schema objects in a `$ref` value that is in an Example Object. Did I get this correctly?

Reply:
Makes sense :) Just thinking about if it's more valuable for the users to have a generic autocompletion for all possible reference values (within the document).

Reply:
Might be, I'll test this locally in the near future and see how it would look 👍